articles/laravel-language-localisation/index.md

```diff
@@ -237,15 +201,15 @@ Next step, open the ``` 'language.php' ``` file in your text editor for Spanish
  ?>
 
  ```
-Pay attention to what's going on in these two files,  
+Pay attention to what's going on in these two files.
 * Filename to hold strings are the same.
 * The array element names are the same for files.
 * The assignment strings have been translated to respective languages, in this case, Espanol and English.
 
 If you're comfortable with whats we have learned so far, it's time to make this simple application into action. 
```

Comment:
kindly, check rewrite this sentence

Reply:
I don't understand,can you  at least be specific on what I should change?

Reply:
@WanjaMIKE I've checked and I could'nt see any reason to modify, the whole meaning will be lost, unless I remove the entire paragraph or simply add a colon at the end of this sentence if that's what you mean as follows```Pay attention to what's going on in these two files:-```, otherwise be specific.Thank You!

Reply:
please rewrite this
If you're comfortable with whats we have learned so far, it's time to make this simple application into action.

Reply:
@WanjaMIKE  done.